src/main/java/org/terasology/structureTemplates/internal/systems/AddItemsToChestSystem.java

```diff
@@ -48,6 +49,12 @@
     @In
     private InventoryManager inventoryManager;
 
+    /**
+     * Listens to structures spawning. When a structure spawns, all of chests specified in the prefab get filled with the specified items through @link addItemToChest.
+     * @param event The event passed.
+     * @param component The component as provided in the prefab. Contains the items, location and such.
+     * @param entity A wrapper around an entity id providing access to common functionality.
+     */
```

Comment:
The method is only public so that it can work with the `@ReceiveEvent annotiation` properly. It is not for use and thus doesn't really need javadoc.

The meaning of the arguments gets already defined by the `@ReceiveEvent` annotation. Every method annotated with that contains the event it handles as first argument and the affected entity as second argument. Any further component arguments are components the entity must have for this event handler to be called.

I don't think we should add this javadoc as it does not provide extra value. Javadoc on methods that don't get called from outside should really just inform about important facts that would surprise another developer that just haves a quick glance at the method.